releasenotes/notes/transpile_init_c-89e73e83589c3c36.yaml

```
@@ -0,0 +1,7 @@
+---
+features_c:
+  - |
+    Added a new function :cpp:func:`qk_transpile_stage_init` to the C API.
```

C linking works now, so you can replace this back to :c:
```suggestion
    Added a new function :c:func:`qk_transpile_stage_init` to the C API.
```